### ISSUE TITLE
Is development on this plugin dead?

### ISSUE THREAD

**Issue by LukaszJaro** [sendgrid/wordpress#87]

**Comment by carl-alberto** [comment]
Yeah, I was wondering who now maintains this plugin, checking the top contributors here https://github.com/sendgrid/wordpress/graphs/contributors and pinging @danielamihalache @vladiacobsendgrid @laur-craciun @radusg hopefully one of them responds and know someone might me maintaining this plugin or they might help us to the right direction

**Comment by isaaclubow** [comment]
SendGrid support has disavowed all knowledge of this plugin and even went so far yesterday as to claim that "WordPress controls it" and that they are neither the authors nor maintainers. Kind of flabbergasting considering WP powers a quarter of the Internet. Considering a fork if they're going to be so obtuse about it...

**Comment by carl-alberto** [comment]
Hmm, weird, Also looking forward for a fork of this project if that is the case :)

**Comment by RobK410** [comment]
> SendGrid support has disavowed all knowledge of this plugin and even went so far yesterday as to claim that "WordPress controls it" and that they are neither the authors nor maintainers. Kind of flabbergasting considering WP powers a quarter of the Internet. Considering a fork if they're going to be so obtuse about it...

This is pretty odd considering as of today SendGrid's documentation and integration providers both show support for a WordPress plugin integration.
